testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -183,6 +183,11 @@ public void testViewReferencingTableInDifferentSchema()
         onHive().executeQuery(format("CREATE VIEW %s.%s AS SELECT * FROM %s.%s", schemaX, viewName, schemaY, tableName));
 
         assertThat(onTrino().executeQuery(format("SELECT COUNT(*) FROM %s.%s", schemaX, viewName))).containsOnly(row(25));
+
+        onHive().executeQuery(format("DROP TABLE %s.%s", schemaY, tableName));
+        onHive().executeQuery(format("DROP VIEW %s.%s", schemaX, viewName));
+        onHive().executeQuery(format("DROP SCHEMA %s", schemaX));
```

Comment:
Is `DROP SCHEMA` actually not enough?

Reply:
it fails if there are tables/views inside, I'd need to force it using `DROP SCHEMA xxx CASCADE`, maybe it would be better I don't have an opinion regarding that